src/utils/slack.js

```diff
@@ -1,41 +1,28 @@
 const token = process.env.SLACK_TOKEN;
 
-const usernameToID = async username => {
-	let members;
-	await fetch(`https://slack.com/api/users.list?token=${token}`)
+const getUserSlackID = (username, users) => {
+	const user = users.find(
+		user => user.name === username || user.profile.real_name === username
+	);
+	return user ? user.id : null;
+};
+
+const loadUsers = async callback => {
```

Comment:
We could take advantage of fetch returning a promise to avoid using callbacks, e.g.

```
export const loadUsers = () =>
  fetch(...)
    .then(response => response.ok ? response.json() : Promise.resolve(new Error('...')))
    .then(data => data.members)

// Component

this.props.loadUsers()
  .then(users => this.setState({ users })
  .catch(loadUsersError => this.setState({ loadUsersError }))
```